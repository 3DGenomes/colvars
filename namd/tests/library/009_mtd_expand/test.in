colvarstrajfrequency    1
colvarsrestartfrequency 10

colvar {
    name d

    hardLowerBoundary yes
    lowerBoundary 0
    upperBoundary 10
    width 0.2
    expandBoundaries yes

    distance {
        group1 { atomnumbers 4 }
        group2 { atomnumbers 99 }
    }
}

metadynamics {
    colvars        d
}
