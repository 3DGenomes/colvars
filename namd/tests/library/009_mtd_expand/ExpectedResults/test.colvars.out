colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013 in any publication based on this calculation.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # colvarsTrajAppend = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = d
colvars:   Initializing a new "distance" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:       Initializing atom group "group1".
colvars:       # scalable = off [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group1" defined, 1 atoms initialized: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # scalable = off [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group2" defined, 1 atoms initialized: total mass = 12.011, total charge = 0.07.
colvars:   All components initialized.
colvars:   # width = 0.2
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 10
colvars:   # upperWallConstant = 0 [default]
colvars:   # hardLowerBoundary = on
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = on
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "metadynamics" instance.
colvars:   # name = "metadynamics1" [default]
colvars:   # colvars = { d }
colvars:   # outputEnergy = off [default]
colvars:   # hillWeight = 0.01 [default]
colvars:   # newHillFrequency = 1000 [default]
colvars:   # hillWidth = 1.25331 [default]
colvars:   # multipleReplicas = off [default]
colvars:   # useGrids = on [default]
colvars:   # gridsUpdateFrequency = 1000 [default]
colvars:   # rebinGrids = off [default]
colvars:   Metadynamics bias "metadynamics1": Will expand grids when the colvar "d" approaches its boundaries.
colvars:   # keepHills = off [default]
colvars:   # writeFreeEnergyFile = on [default]
colvars:   # saveFreeEnergyFile = off [default]
colvars:   # writeHillsTrajectory = off [default]
colvars:   # wellTempered = off [default]
colvars:   # biasTemperature = -1 [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (2 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group d:0/0. 1 atoms: total mass = 12.011.
colvars: Re-initialized atom group d:0/1. 1 atoms: total mass = 12.011.
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: Updating NAMD interface:
colvars: updating atomic data (2 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars:   Metadynamics bias "metadynamics1": new upper boundary for colvar "d", at 15.2.
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
